service/README.md

```diff
@@ -53,17 +53,87 @@ Change the variable for CCD_DEF_CASE_SERVICE_BASE_URL in bin/configurer/utils/fp
 the service.
 
 ### Run the application (from IntelliiJ)
- 
-Ensure the Spring Boot is started with local profile 
-(add environment variable spring.profiles.active=local when starting the main class).
 
-Configure the notify.api_key if necessary (you can pass them as environment variables when IntelliJ 
-starts the application). 
+Ensure the Spring Boot is started with local, user-mappings and the feature-toggle profiles
+(add environment variable spring.profiles.active=feature-toggle,local,user-mappings when starting the main class).
 
-Ensure the variable CCD_DEF_CASE_SERVICE_BASE_URL bin/configurer/utils/fpl-process-definition.sh is set to
-http://docker.for.mac.localhost:4000 - this will use the host machine for CCD, reaching 
-locally running application. 
+Configure the notify.api_key if necessary (you can pass them as environment variables when IntelliJ
+starts the application).
 
+Ensure the variable CCD_DEF_CASE_SERVICE_BASE_URL bin/configurer/utils/fpl-process-definition.sh is set to
+http://docker.for.mac.localhost:4000 - this will use the host machine for CCD, reaching
+locally running application.
+
+## Config
+
+Services uses three sources of configuration for the placeholders:
+* application.yaml (profile-driven, test and production systems use only default profile)
+* ENV variables (passed in via charts in the cluster or docker-compose for local)
+* Secrets from Azure Key vaults and following [library](https://github.com/hmcts/properties-volume-spring-boot-starter).
+  The secret names are map to the properties as specified in [this file](src/main/resources/bootstrap.yaml)
+
+Custom configuration parameters:
+
+|Property name|Configuration place|Description|
+|---|---|---|
+|fees-register.api.url|ENV|Address of the Fees Register service, used for obtaining fee of applications|
```

Comment:
Everywhere else you have mentioned "URL of service x" instead of "Address of service x", probably good to be consistent

Reply:
Changed to URL.